lib/global_id/locator.rb

```diff
@@ -139,6 +126,26 @@ class ActiveRecordFinder
         def locate(gid)
           gid.model_class.find gid.model_id
         end
+
+        def locate_many(gids, options = {})
+          models_and_ids  = gids.collect { |gid| [ gid.model_class, gid.model_id ] }
+          ids_by_model    = models_and_ids.group_by(&:first)
+          loaded_by_model = Hash[ids_by_model.map { |model, ids|
+            [ model, find_records(model, ids.map(&:last), ignore_missing: options[:ignore_missing]).index_by { |record| record.id.to_s } ]
```

Comment:
I'm not sure if we should push the gids manipulation up in the stack and leave only `find_records` here - the grouping logic is here because we can't retrieve different classes in the same query, which might not be true with a custom locator - but I don't have a real example for this scenario.